torchtitan/datasets/hf_datasets.py

```diff
@@ -19,49 +18,39 @@
 from datasets import Dataset, load_dataset
 from datasets.distributed import split_dataset_by_node
 
-# map from dataset name to a local directory, or
-# a dataset repository on the HF hub
+# To load your own custom dataset, please follow instructions in docs/datasets.md
+
+
+def load_c4_dataset(dataset_path: str):
+    """Load C4 dataset with default configuration."""
+    logger.info("Loading C4 dataset...")
```

Comment:
let's remove this redundant logging by default; users can always have their own logging.